src/script/view_model/CallingViewModel.ts

```diff
@@ -240,10 +240,9 @@ export class CallingViewModel {
       this.callingRepository.setEpochInfo(conversationId, {epoch, keyLength, secretKey}, members);
     };
 
-    const leaveCall = async (conversationId: QualifiedId) => {
-      const conversation = this.getConversationById(conversationId);
-
-      if (!conversation?.isUsingMLSProtocol) {
+    const closeCall = async (conversationId: QualifiedId, conversationType: CONV_TYPE) => {
+      // There's nothing we need to do for non-mls calls
+      if (conversationType !== CONV_TYPE.CONFERENCE_MLS) {
```

Comment:
In case of closing a call after deleting a conversation, the conversation might not be found with `getConversationById` as call close works asynchronously (and is fired by avs). So it's better to relay on a call conversation type.